Library/Homebrew/utils/shared_audits.rb

```diff
@@ -59,7 +60,8 @@ def github_release(user, repo, tag, formula: nil, cask: nil)
     end
 
     if release["prerelease"]
-      return if formula && GITHUB_PRERELEASE_ALLOWLIST[formula.name] == formula.version
+      return if formula && (GITHUB_PRERELEASE_ALLOWLIST[formula.name] == formula.version ||
+                            GITHUB_PRERELEASE_ALLOWLIST[formula.name] == :all)
```

Comment:
This should probably also fail if it's not a pre-release anymore but still in the `GITHUB_PRERELEASE_ALLOWLIST` like above.